.travis.yml

```diff
@@ -1,8 +1,16 @@
 language: go
 sudo: false
+arch:
+  - AMD64
+  - ppc64le
 go:
   - 1.4
   - 1.5
   - 1.6
   - tip
+# Disable version 1.4
```

Comment:
For context, why is is 1.4 specifically singled out here?

Overall, all of these are ancient version anyway, so I'd be ok with dropping the oldest ones and just keep 1.6.

Reply:
yeah ok, please go ahead dropping the oldest ones and keep 1.6. Thanks

Reply:
Feel free to just amend this PR, dropping 1.4 & 1.5 from the list above instead of adding this `exclude` entry.

Reply:
Dropped 1.4 & 1.5 Go versions and deleted exclude jobs entry too....

Reply:
If agreed with the current changes, please accept the build & resolve the conversation status